src/Cliente.ts

```diff
@@ -0,0 +1,95 @@
+
+import { Usuario } from './Usuario';
+export class Cliente extends Usuario{
+
+
+    edad: number;
+    readonly genero: boolean;
+    altura: number;
+    private peso: number;
+    private cintura: number;
+    private cuello: number;
+    private idCliente: number;
+
+
+    /**
+     * Crea un objeto de la clase Cliente.
+     * @param idCliente: Identificador del cliente.
+     * @param nombre: Nombre del cliente de tipo string.
+     * @param primerApellido: Primer apellido de tipo string.
```

Comment:
No me creo que uséis los dos apellidos de forma diferenciada en la lógica de negocio. Pero si @JaimeGM96 lo ha dicho, ¿quién soy yo para corregirlo?